src/ui/public/kbn_top_nav/kbn_top_nav.js

```diff
@@ -45,10 +46,6 @@ module.directive('kbnTopNav', function (Private) {
     restrict: 'E',
     transclude: true,
     template($el, $attrs) {
```

Comment:
Word, sexy, I don't think this needs to be in a function like this any longer. I only had it as such for the `navbar-extensions` directive